crates/api_models/src/enums.rs

```diff
@@ -394,11 +394,12 @@ pub enum FieldType {
     UserAddressCity,
     UserAddressPincode,
     UserAddressState,
-    UserAddressCountry,
+    UserAddressCountry { options: Vec<String> },
```

Comment:
Why is this field added? I see that `UserCountry` is also present, if `UserCountry` is not needed remove that enum variant

Reply:
So there is country inside the payment_method_data which is UserCountry, and for country inside Address struct is UserAddressCountry